packages/plugins/graphql-sse/src/index.ts

```diff
@@ -0,0 +1,75 @@
+import { getOperationAST } from 'graphql'
+import { Plugin, YogaInitialContext } from 'graphql-yoga'
+import { HandlerOptions } from 'graphql-sse'
+import { createHandler, RequestContext } from 'graphql-sse/lib/use/fetch'
+
+export interface GraphQLSSEPluginOptions
+  extends Omit<
+    HandlerOptions<Request, RequestContext, any>,
+    'validate' | 'execute' | 'subscribe' | 'schema' | 'onSubscribe'
+  > {
+  /**
+   * Endpoint location where GraphQL over SSE will be served.
+   *
+   * @default '/graphql/stream'
+   */
+  endpoint?: string
+}
+
+/**
+ * Get [GraphQL over Server-Sent Events Protocol](https://github.com/enisdenjo/graphql-sse/blob/master/PROTOCOL.md) integration with GraphQL Yoga by simply installing this plugin!
+ *
+ * Note that the endpoint defaults to `/graphql/stream`, this is where your [graphql-sse](https://github.com/enisdenjo/graphql-sse) client should connect.
+ */
+export function useGraphQLSSE(
+  options: GraphQLSSEPluginOptions = {},
+): Plugin<YogaInitialContext> {
+  const { endpoint = '/graphql/stream', ...handlerOptions } = options
+  let handler!: (request: Request) => Promise<Response>
+  return {
+    onYogaInit({ yoga }) {
+      handler = createHandler(
+        {
+          ...handlerOptions,
+          async onSubscribe(req, params) {
+            const enveloped = yoga.getEnveloped({
+              // TODO: serverContext
```

Comment:
Any more context on this TODO?

Reply:
https://github.com/dotansimha/graphql-yoga/pull/2240/commits/6b74a3d388f7084619f633c23cd040bac1c8d764